### PR TITLE
Basic Swift pack with example

### PR DESCRIPTION
**Pull request by nomisbeme**
Simple Swift 3 pack based on the php example.

Feedback and suggestions appreciated.

### CODE REVIEW

File: packs/swift/chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Default values for swift.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 2
+image:
+  registry: docker.io
+  org: nomisbeme
+  name: swift-onbuild-bitnami
+  tag: 3.0.2.1
+  pullPolicy: Always
```
Comment (bacongobbler) [inline packs/swift/chart/values.yaml]: this should be changed to `ifNotPresent` as seen [in other charts](https://github.com/Azure/draft/blob/master/packs/golang/chart/values.yaml#L10).

**Comment by bacongobbler** [comment]
closes #96

**Comment by nomisbeme** [comment]
Thanks @bacongobbler, good catch.

**Comment by bacongobbler** [comment]
Would you be amenable to removing the NOTICE file in `packs/swift`? I'm just checking with our legal team for advice, but I'm pretty sure that because the code is your own original contribution and you've signed the CLA, you are giving Microsoft

> a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third parties.

According to [Section 5a. of the CLA](https://cla.microsoft.com/cladoc/microsoft-contribution-license-agreement.pdf). The only reason why we have NOTICES in other packs is because we used the [Heroku Buildpack detect scripts](https://github.com/heroku/heroku-buildpack-python/blob/master/bin/detect).

I'm new to MSFT hence why I'm checking with the legal team, but just thought you should know.

**Comment by bacongobbler** [comment]
Would you mind fixing the failing unit test?

**Comment by bacongobbler** [comment]
closing in favour of #189

**Pull request closed without merging**